2022-02-06 01:45:27,849:INFO:Reading config parameters
2022-02-06 01:45:27,850:INFO:Reading config parameters completed
2022-02-06 01:45:27,850:INFO:Function create_dim_date_df() running 
2022-02-06 01:45:27,926:INFO:Database connection success
2022-02-06 01:45:32,980:INFO:Reading config parameters
2022-02-06 01:45:32,980:INFO:Reading config parameters completed
2022-02-06 01:45:32,981:ERROR:Error -> [Errno 2] No such file or directory: '../source_data/cities.csv'
Traceback (most recent call last):
  File "load_dim_location.py", line 32, in <module>
    df_city = pd.read_csv('../source_data/cities.csv')
  File "C:\Users\nishe\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\io\parsers.py", line 610, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "C:\Users\nishe\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\io\parsers.py", line 462, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "C:\Users\nishe\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\io\parsers.py", line 819, in __init__
    self._engine = self._make_engine(self.engine)
  File "C:\Users\nishe\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\io\parsers.py", line 1050, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "C:\Users\nishe\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\io\parsers.py", line 1867, in __init__
    self._open_handles(src, kwds)
  File "C:\Users\nishe\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\io\parsers.py", line 1368, in _open_handles
    storage_options=kwds.get("storage_options", None),
  File "C:\Users\nishe\AppData\Local\Programs\Python\Python37\lib\site-packages\pandas\io\common.py", line 647, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: '../source_data/cities.csv'
2022-02-06 01:46:13,278:INFO:Reading config parameters
2022-02-06 01:46:13,279:INFO:Reading config parameters completed
2022-02-06 01:46:13,360:INFO:Database connection success
2022-02-06 01:46:18,926:INFO:Reading config parameters
2022-02-06 01:46:18,926:INFO:Reading config parameters completed
2022-02-06 01:46:19,076:INFO:Database connection success
2022-02-06 01:46:19,126:INFO:Database connection success
